lib/netext/grpcext/conn.go

```diff
@@ -162,8 +162,11 @@ func (c *Conn) Invoke(
 		// {"x":6,"y":4,"z":0}
 		raw, _ := marshaler.Marshal(resp)
 		msg := make(map[string]interface{})
-		_ = json.Unmarshal(raw, &msg)
+		er := json.Unmarshal(raw, &msg)
```

Comment:
nit: you should be able to assign to `err` without `:=`  and without getting a complain about shadowing

Reply:
I don't know what you're talking about . Changing the variable name solves the shading problem. The `err` variable already exists. What exactly did I do wrong?

Reply:
You should just assign this to `err` instead of addng a new variable.

I don't se ay reason not to.